app/controllers/users_controller.rb

```diff
@@ -100,6 +113,22 @@ def authenticate_sendgrid_webhook
   def emit_lesson_notification_on_socket(lesson)
     return unless Rails.env.production?
 
+    # Notify iOS apps
+    # Swap lines below when production certificate is ready
+    #myAPN = Houston::Client.production
+    myAPN = Houston::Client.development
+    myAPN.certificate = IOS_CERTIFICATE
+    myAPN.passphrase = IOS_CERTIFICATE_PASSPHRASE
+
+    DeviceToken.all.each do |dt|
+      notification = Houston::Notification.new(device: dt.token)
+      notification.alert = lesson.title
+      notification.sound = 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.